docs-ref-services/latest/eventgrid.md

```diff
@@ -31,26 +31,27 @@ To manage your Azure Event Grid resources like Event Grid instances, topics, and
 
 Create events, authenticate, and post to topics using the below packages
 
-### v2 of @azure/eventgrid
+### v4 of @azure/eventgrid
 
-To create events, authenticate, and post to topics, you would use the `@azure/eventgrid` package.
+To create events, authenticate, and post to topics, we recommend you use the latest version of the `@azure/eventgrid` package.
 
 | NPM Package                                                        | Reference                                                                                            |   
 | ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- | 
-| [@azure/eventgrid v2](https://npmjs.com/package/@azure/eventgrid) | [API Reference for @azure/eventgrid v2](https://docs.microsoft.com/javascript/api/@azure/eventgrid) | 
+| [@azure/eventgrid v4](https://www.npmjs.com/package/@azure/eventgrid) | [API Reference for @azure/eventgrid v3](https://docs.microsoft.com/javascript/api/@azure/eventgrid/) |
 
-### v3 of @azure/eventgrid (in preview)
 
-To create events, authenticate, and post to topics, you can also try latest version of the `@azure/eventgrid` i.e. v3. Since this version is still in preview, we do not recommend using this in your production environment until we release a stable version.
+### v2 of @azure/eventgrid (legacy)
+
+To create events, authenticate, and post to topics, you can use the older `@azure/eventgrid` packageas well. Since this version will only receive critical bug fixes and no new features, we recommend using the v4 version `@azure/eventgrid` instead.
 
 | NPM Package                                                        | Reference                                                                                            |   
 | ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- | 
-| [@azure/eventgrid v3](https://www.npmjs.com/package/@azure/eventgrid/v/next) | [API Reference for @azure/eventgrid v3](https://docs.microsoft.com/en-us/javascript/api/@azure/eventgrid/?view=azure-node-preview) | 
+| [@azure/eventgrid v2](https://www.npmjs.com/package/@azure/eventgrid/v/2.0.0) | [API Reference for @azure/eventgrid v2](https://docs.microsoft.com/javascript/api/@azure/eventgrid/?view=azure-node-legacy) |
 
 
 ### v1 of azure-eventgrid
```

Comment:
How about collapsing this with the previous section? We can have the header as `Legacy packages` and then the table can contain 2 rows. 1 for `@azure/eventgrid` v2 and the other for `azure-eventgrid`.

Reply:
Sounds good to me!